src/maturin/springs.clj

```diff
@@ -0,0 +1,62 @@
+;; create xs
+(defn gen-xs [n]
```

Comment:
Looks great!! You might take the namespace definition from `better` , so all the imports are here and working. I'll pull and get the animation running myself this eve!

Reply:
Thanks! Added.